src/xo-app/user/index.js

```diff
@@ -55,8 +55,6 @@ const FILTER_TYPE_TO_LABEL_ID = {
   vmTemplate: 'homeTypeVmTemplate'
 }
 
-const SSH_KEY_STYLE = { wordWrap: 'break-word' }
```

Comment:
You might need to use it in the SortedTable. Have you tested it with an actual SSH key to see how it looks?

Reply:
I don't know when i can include this style.

Reply:
```js
itemRenderer: sshKey => <span style={SSH_KEY_STYLE}>
  {sshKey.key}
</span>
```